Formula/easyengine.rb

```diff
@@ -27,6 +27,8 @@ def install
   end
 
   test do
+    return if OS.linux? # requires `sudo`
+
     system bin/"ee config set locale hi_IN"
```

Comment:
Let's clean this line up too, please.

Reply:
See patch above.

Reply:
Sorry, I had missed that. Updating.

Reply:
@carlocab Done with the changes. Anything else?